proto/proto-backplane-grpc/src/main/proto/deephaven/proto/ticket.proto

```diff
@@ -23,4 +23,13 @@ option optimize_for = SPEED;
  */
 message Ticket {
     bytes ticket = 1;
+}
+
+message TypedTicket {
+    bytes ticket = 1;
```

Comment:
Do you have a strong preference against nesting Ticket here? Nesting it makes it easier imho to actually handle the object that pops out as a plain Ticket, given all the wiring that already exists to handle ticket objects, without unwrapping+re-wrapping each `bytes` as you hit it

Reply:
No strong preference, I can switch.

Reply:
Ill regen on my side - everything except fetching tables is done for the js api changes, i'll get you those patches ~monday~ tuesday.